tests/integration/components/sl-chart-test.js

```diff
@@ -0,0 +1,117 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-chart', 'Integration | Component | sl chart', {
+    integration: true
+});
+
+test( 'Defaults applied correctly', function( assert ) {
+    this.set( 'testseries', []);
+    this.set( 'testoptions', {});
+
+    this.render( hbs`
+      {{#sl-chart series=testseries options=testoptions}}
+      {{/sl-chart}}
+    ` );
```

Comment:
Aziz is out of the office this week and next week but before he left I mentioned to him that his tests were failing linting and we agreed that he needed to update his block statements to follow the convention that I set. Aziz mentioned that the would submit a PR to make those fixes but I suspect he ran out of time before and did not get it in before he left for vacation.